cmd/main.go

```diff
@@ -66,8 +66,8 @@ func main() {
 	switch {
 	case jsonlFlag:
 		dataHandler = handlers.JSONLHandler{}
-	// case csvFlag:
-	// 	dataHandler = handlers.CSVHandler{}
+	case csvFlag:
+		dataHandler = handlers.CSVHandler{}
```

Comment:
I make this change in #83. Once I complete #84, I plan on merging that

https://github.com/kevmo314/appendable/pull/83/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R68-R69

Reply:
Oh nice!